### ISSUE TITLE
Excuse me, how to freeze the layer of loading parameters after loading model parameters?

### ISSUE THREAD

**Issue by ly19970621** [open-mmlab/mmsegmentation#521]
After I ran according to the configuration file corresponding to the table, I found that the result was generally 10% worse than that of the table, because I only had one GPU. And I want to add a module on the basis of your model. After training for a few days, I found that the improvement is very little, and there is still a big gap from the accuracy of the original model in the table. So I want to load the model parameters that you have trained, freeze the layers of these loading parameters, and only train the modules that I have added. Excuse me, is this method feasible? How to implement this method?

**Comment by yitianlian** [comment]
Could you say more details about which file should I add <requires_grad=False>? My backbone is MAE, which didn't support "frozen_stages=12".Can you help me?

**Comment by xiexinch** [comment]
> Could you say more details about which file should I add <requires_grad=False>? My backbone is MAE, which didn't support "frozen_stages=12".Can you help me?

Hi @yitianlian,
Sorry for the late reply, here is an example about adding `frozen_stages`.
https://github.com/open-mmlab/mmsegmentation/blob/43b4efb122f1c4e934ee2588f40210e8c34eed5f/mmseg/models/backbones/swin.py#L642
And `requires_grad` is an attribute of `torch.Tensor`.
https://pytorch.org/docs/stable/generated/torch.Tensor.requires_grad.html#torch.Tensor.requires_grad
